rust-toolchain

```diff
@@ -1,6 +1,8 @@
 [toolchain]
-# N.B.: Update .github and .circleci yaml to use a matching image.
-channel = "1.72.1"
+# N.B.: Update .github and .circleci yaml to use a matching imagae, if available.
```

Comment:
```suggestion
# N.B.: Update .github and .circleci yaml to use a matching image, if available.
```

Reply:
Fixed. The number of trees I burn by being a poor typist is discouraging.